### ISSUE TITLE
Since psutil was removed, we get a module exception at runtime

### ISSUE THREAD

**Issue by TJM** [jduncan-rva/newrhelic#10]
The python-psutil package that comes with EL6: python-psutil-0.6.1-1.el6.x86_64 ... does not support `net_io_counters`

```
>>> n.add_to_newrelic()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 447, in add_to_newrelic
    self._prep_first_run()  #prime the data buffers if it's the first loop
  File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 442, in _prep_first_run
    raise e
AttributeError: 'module' object has no attribute 'net_io_counters'
```

In addition to that, I found another error where one of us mis-spelled exception (oops):

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "test.py", line 3, in <module>
    n.add_to_newrelic()
  File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 429, in add_to_newrelic
    self._prep_first_run()  #prime the data buffers if it's the first loop
  File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 423, in _prep_first_run
    logging.excption(e)
AttributeError: 'module' object has no attribute 'excption'
```

... the fix for "excption" will be coming in Issue #9

**Comment by jduncan-rva** [comment]
ah. here's an issue. psutil 0.6.x doesn't have the net_io_counters
function. Just downloaded the srpm from EPEL and confirmed.

On Thu, Jun 12, 2014 at 3:42 PM, Tommy McNeely notifications@github.com
wrote:

> The python-psutil package that comes with EL6:
> python-psutil-0.6.1-1.el6.x86_64 ... does not support net_io_counters
> 
> > > > n.add_to_newrelic()
> > > > Traceback (most recent call last):
> > > >   File "<stdin>", line 1, in <module>
> > > >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 447, in add_to_newrelic
> > > >     self._prep_first_run()  #prime the data buffers if it's the first loop
> > > >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 442, in _prep_first_run
> > > >     raise e
> > > > AttributeError: 'module' object has no attribute 'net_io_counters'
> 
> In addition to that, I found another error where one of us mis-spelled
> exception (oops):
> 
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "test.py", line 3, in <module>
>     n.add_to_newrelic()
>   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 429, in add_to_newrelic
>     self._prep_first_run()  #prime the data buffers if it's the first loop
>   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 423, in _prep_first_run
>     logging.excption(e)
> AttributeError: 'module' object has no attribute 'excption'
> 
> ... the fix for "excption" will be coming in Issue #9
> https://github.com/jduncan-rva/newRHELic/issues/9
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jduncan-rva/newRHELic/issues/10.

## 

Thanks,

Jamie Duncan
@jamieeduncan

**Comment by jduncan-rva** [comment]
scratch that. was looking in the wrong place.

On Thu, Jun 12, 2014 at 4:28 PM, Jamie Duncan jamie.e.duncan@gmail.com
wrote:

> ah. here's an issue. psutil 0.6.x doesn't have the net_io_counters
> function. Just downloaded the srpm from EPEL and confirmed.
> 
> On Thu, Jun 12, 2014 at 3:42 PM, Tommy McNeely notifications@github.com
> wrote:
> 
> > The python-psutil package that comes with EL6:
> > python-psutil-0.6.1-1.el6.x86_64 ... does not support net_io_counters
> > 
> > > > > n.add_to_newrelic()
> > > > > Traceback (most recent call last):
> > > > >   File "<stdin>", line 1, in <module>
> > > > >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 447, in add_to_newrelic
> > > > >     self._prep_first_run()  #prime the data buffers if it's the first loop
> > > > >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 442, in _prep_first_run
> > > > >     raise e
> > > > > AttributeError: 'module' object has no attribute 'net_io_counters'
> > 
> > In addition to that, I found another error where one of us mis-spelled
> > exception (oops):
> > 
> > Traceback (most recent call last):
> >   File "<stdin>", line 1, in <module>
> >   File "test.py", line 3, in <module>
> >     n.add_to_newrelic()
> >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 429, in add_to_newrelic
> >     self._prep_first_run()  #prime the data buffers if it's the first loop
> >   File "/home/tmcneely/Projects/newrhelic/NewRHELic/lib/python2.6/site-packages/newrhelic.py", line 423, in _prep_first_run
> >     logging.excption(e)
> > AttributeError: 'module' object has no attribute 'excption'
> > 
> > ... the fix for "excption" will be coming in Issue #9
> > https://github.com/jduncan-rva/newRHELic/issues/9
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/jduncan-rva/newRHELic/issues/10.
> 
> ## 
> 
> Thanks,
> 
> Jamie Duncan
> @jamieeduncan

## 

Thanks,

Jamie Duncan
@jamieeduncan
